src/separations_tests.cc

```diff
@@ -43,6 +43,199 @@ TEST(SeparationsTests, SepMaterial) {
   EXPECT_DOUBLE_EQ(0, mqsep.mass("Am242"));
 }
 
+  
+// Test if a total separation of 1 and less works as expected.
```

Comment:
"Check that cumulative separations efficiency for a single nuclide of less than or equal to one does not trigger an error."